launch/client.py

```diff
@@ -630,36 +661,39 @@ def create_model_endpoint(
             # Presumably, the user knows that the endpoint doesn't already exist, and so we can defer
             # to the server to reject any duplicate creations.
             logger.info("Creating new endpoint")
-            bundle_name = _model_bundle_to_name(model_bundle)
-            payload = dict(
-                endpoint_name=endpoint_name,
-                bundle_name=bundle_name,
-                cpus=cpus,
-                memory=memory,
-                storage=storage,
-                gpus=gpus,
-                gpu_type=gpu_type,
-                min_workers=min_workers,
-                max_workers=max_workers,
-                per_worker=per_worker,
-                endpoint_type=endpoint_type,
-                post_inference_hooks=post_inference_hooks,
-                labels=labels or {},
-            )
-            if gpus == 0:
-                del payload["gpu_type"]
-            elif gpus > 0 and gpu_type is None:
```

Comment:
looks like we're removing a bit of client-side validation but that's probably not a big deal

Reply:
yeah oops - though I think there's server-side validation for this now